RELEASE_NOTES.md

```diff
@@ -1,5 +1,16 @@
 # Release Notes
 
+## 4.6.1
```

Comment:
I'm not sure the changes in this file should be going to the main branch

Reply:
I think we usually try to keep main updated with all the release notes. https://github.com/containers/podman/blob/6ab38e55e733e70c9592ce41132a0f390b236560/RELEASE_PROCESS.md?plain=1#L222